### ISSUE TITLE
elm-factory build produces error

### ISSUE THREAD

**Issue by chmar77** [farism/elm-factory#4]
Output:
```
elm-factory build
/usr/local/lib/node_modules/elm-factory/node_modules/yargs/yargs.js:1079
      else throw err
           ^

Error: Task build-html is not configured as a task on gulp.  If this is a submodule, you may need to use require('run-sequence').use(gulp).
    at /usr/local/lib/node_modules/elm-factory/node_modules/run-sequence/index.js:23:10
    at Array.forEach (native)
    at verifyTaskSets (/usr/local/lib/node_modules/elm-factory/node_modules/run-sequence/index.js:16:11)
    at runSequence (/usr/local/lib/node_modules/elm-factory/node_modules/run-sequence/index.js:112:2)
    at Gulp.gulp.task (/usr/local/lib/node_modules/elm-factory/src/tasks/build.js:96:5)
    at module.exports (/usr/local/lib/node_modules/elm-factory/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/usr/local/lib/node_modules/elm-factory/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/usr/local/lib/node_modules/elm-factory/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/usr/local/lib/node_modules/elm-factory/node_modules/orchestrator/index.js:134:8)
    at Object.handler (/usr/local/lib/node_modules/elm-factory/bin/cmds/build.js:14:10)
```

**Comment by farism** [comment]
Thank you for the bug report.  The error should be fixed with `0.3.1` which I just now published to npm. Could you please confirm if it is fixed on your end?

Also, generation of the actual html file is temporarily disabled until I can get the revisioning working with the manifests.

**Comment by farism** [comment]
I have released two versions since this issue and have had other people confirm this working, so closing for now. Feel free to add any additional feedback if this still persists.
